module/x/gravity/client/cli/tx.go

```diff
@@ -128,7 +128,7 @@ func CmdRequestBatchTx() *cobra.Command {
 			}
 
 			denom := args[0]
-			signer, err := sdk.AccAddressFromHex(args[1])
+			signer, err := sdk.AccAddressFromHexUnsafe(args[1])
```

Comment:
why is it unsafe?

Reply:
maybe it can't tell if it's actually an address or some random 20 bytes